src/lib/components/ui/Modal/Modal.jsx

```diff
@@ -152,6 +190,7 @@ class Modal extends React.Component {
 
 Modal.defaultProps = {
   actions: [],
+  autoFocusDisabled: false,
```

Comment:
I'd prefer to turn it around: `autoFocus: false,`

Reply:
I am somewhere between true and false as default. But I would say we want to have it enabled by default, because there is no reason to let it off. I will always behave better by default. And if someone want to control it manually that it can be disabled.

Reply:
@bedrich-schindler I was rather preferring `false` but your last sentence convinced me.

Reply:
I wish someone would do the thinking for me…

Reply:
@mbohal In general? Or just here? 🙂

Reply:
@adamkudrna in general, i find it hard and exhausting